cms/templates/js/certificate-details.underscore

```diff
@@ -0,0 +1,40 @@
+<div class="collection-details wrapper-certificate">
+    <header class="collection-header">
+        <h3 class="title">
+            <a href="#" class="toggle detail-toggle <% if (showDetails){ print('hide'); } else { print('show'); } %>-details">
+                <i class="ui-toggle-expansion icon fa fa-caret-<% if (showDetails){ print('down'); } else { print('right'); } %>" aria-hidden="true"></i>
+                <%= name %>
+            </a>
+        </h3>
+    </header>
+
+   <ol class="collection-info certificate-info certificate-info-<% if(showDetails){ print('block'); } else { print('inline'); } %>">
+     <% if (!_.isUndefined(id)) { %>
+         <li class="certificate-id">
+            <span class="certificate-label"><%= gettext('ID') %>: </span>
+            <span class="certificate-value"><%= id %></span>
+        </li>
+     <% } %>
+     <% if (showDetails) { %>
+        <li class="collection-description certificate-description">
+            <%= description %>
+        </li>
+        <li class="collection-description certificate-description">
+         <header>
+           <span class="title-2"><%= gettext("Course Signatories") %></span>
```

Comment:
use same class as the edit view for certificates (shown below), remove header wrapper from this area.
![image](https://cloud.githubusercontent.com/assets/2023680/7681458/70f64482-fd3e-11e4-8c77-18cd8056c8bc.png)